tests/dummy/app/router.js

```diff
@@ -17,6 +17,7 @@ Router.map(function () {
     this.route('tab')
     // Tests
     this.route('hook')
+    this.route('block-format')
```

Comment:
You will also need to modify the overview page to see the new demo on the left panel.